docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -284,6 +284,31 @@ PUT _ilm/policy/my_policy
 --------------------------------------------------
 // CONSOLE
 
+[[ilm-freeze-action]]
+==== Freeze
+
+Phases allowed: cold.
+
+This action will <<frozen-indices, freeze>> the index
+by calling the <<freeze-index-api, Freeze Index API>>.
```

Comment:
This should have a note similar to the [freeze index API docs](https://www.elastic.co/guide/en/elasticsearch/reference/6.x/freeze-index-api.html) that calls out that this action closes and opens the index, and that has repercussions (index will not be queryable), as that's different from the other actions.